src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
```

Comment:
Extending the `Lottery` class here is not necessary. If you need to use methods from `Lottery`, you should create an instance of the `Lottery` class inside the `main` method and use that instance to call methods.